### PR TITLE
[CI:DOCS] events: document journald identifiers

### PR DESCRIPTION
**Pull request by vrothberg**
#### Does this PR introduce a user-facing change?

```release-note
Document journald identifiers used in the journald events backend.
```

@containers/podman-maintainers PTAL

### CODE REVIEW

**Comment by vrothberg** [comment]
/cherry-pick v4.4

This will help users work with the new "verbose create events".

File: docs/source/markdown/podman-events.1.md [hunk 1]
```diff
@@ -180,6 +180,25 @@ $ podman events --format json
 {"ID":"a0f8ab051bfd43f9c5141a8a2502139707e4b38d98ac0872e57c5315381e88ad","Image":"docker.io/library/alpine:latest","Name":"friendly_tereshkova","Status":"unmount","Time":"2019-04-28T13:43:38.063017276-04:00","Type":"container"}
 ```
 
+## JOURNALD IDENTIFIERS
```
Comment (Luap99) [inline docs/source/markdown/podman-events.1.md:183]: I would move this before the examples section.

**Review by Luap99 — COMMENTED** [review]
on point otherwise LGTM

File: docs/source/markdown/podman-events.1.md (hunk 1 above)
Comment (vrothberg) [reply docs/source/markdown/podman-events.1.md]: Done :heavy_check_mark:

**Pull request merged**